packages/javascript/src/codes.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/naming-convention */
+// This code is automatically generated and should not be manually edited.
+
+/**
+ * There is a proxy configuration, and that configuration specifies to use TLS, but it is not using HTTPS authorization. This is likely not a desired configuration.
+ *
+ * Generate a log string for this code.
+ *
+ * This function will automatically include the log code.
+*/
+export function configuration_usageWarning_proxyTlsAuth(): string {
```

Comment:
I went with a less-elegant code generation style. The nested static classes and methods had a nice interface, but were not suitable for client-side JS because they cannot be tree shaken.

Making each export a function should result in optimal tree-shaking or client-side SDKs.